scripts/test-all.sh

```diff
@@ -62,7 +62,6 @@ RUST_BACKTRACE=1 cargo test \
   --package wasmtime-obj \
   --package wiggle \
   --package wiggle-generate \
-  --package wiggle-runtime \
   --package wiggle-test \
```

Comment:
do you have to add wiggle-macro back here?

Reply:
Sure yeah. TBH I [don't know the fate of this script](https://github.com/bytecodealliance/wasmtime/pull/1397) and it already tests very little compared to CI, but can't hurt to keep touching it.

Reply:
thanks, i hadn't paid attention to all of that discussion.